src/styles/variables.scss

```diff
@@ -28,6 +28,7 @@ $color-grey-dark: #787878;
 $color-grey-border: #757575;
 $color-grey-border-avatar: #9e9e9e;
 $color-purple: #8f00ff;
+$color-grey-dialog: #2b2b2b;
```

Comment:
As the dialogs could have totally different color it makes no sense to use `grey` in its name. So I suggest to rename it to `$color-dialog`.